public/js/user-store.js

```diff
@@ -18,6 +18,10 @@ function UserStore(localDispatcher) {
   localDispatcher.register((function(payload) {
     if (payload.actionType === 'set-user') {
       console.log('UserStore: storing current user:', payload.user);
+      //
+      // Example:
+      // {email: 'f@f.com', is_logged_in: true, payment_methods: []}
```

Comment:
I've been wondering if I should enforce this ``payload.user`` object so it blows up if it's missing some properties. Sigh, types.